packages/localize/src/localize-front-end-module.ts

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
rename localize-frontend-module.ts

Reply:
Done rename the file to be localize-frontend-module.ts